src/web/mjs/connectors/Puzzmos.mjs

```diff
@@ -1,85 +1,12 @@
-import Connector from '../engine/Connector.mjs';
+import MangaReaderCMS from './templates/MangaReaderCMS.mjs';
 
-/**
- * Seems to be customized FlatManga CMS
- * Very similar to EpikManga
- */
-export default class Puzzmos extends Connector {
-
-    /**
-     *
-     */
+export default class Biamam extends MangaReaderCMS {
```

Comment:
Change class name, there is already another connector with that name

Reply:
yes, copy/paste.